scheduler/controllers/odigosconfiguration/odigosconfiguration_controller.go

```diff
@@ -196,6 +196,27 @@ func (r *odigosConfigurationController) handleGoOffsetsCronJob(ctx context.Conte
 		return deleteCronJob(ctx, r.Client, ns, apiVersion)
 	}
 
+	// Determine the mode to use (default to "direct" if not specified)
+	mode := config.GoAutoOffsetsMode
+	if mode == "" {
+		mode = k8sconsts.OffsetCronJobModeDirect
+	}
+
+	// Determine image name and command based on mode
+	var imageName string
+	var command []string
+
+	switch mode {
+	case k8sconsts.OffsetCronJobModeDirect:
+		imageName = k8sconsts.CliImageName
+		command = []string{"pro", "update-offsets"}
+	case k8sconsts.OffsetCronJobModeImage:
+		imageName = k8sconsts.CLIOffsetsImageName
+		command = []string{"pro", "update-offsets", "--from-file", "/odigos/offset_results_min.json"}
```

Comment:
I am probably missing something trivial, but where is the `odigos-cli-offsets` image coming from? I don't see it in the registry or the code for it

Reply:
Ah looks like the push job is failing https://github.com/odigos-io/enterprise-go-instrumentation/pull/1697